yarn.lock

```diff
@@ -0,0 +1,1569 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please remove this file

Reply:
lock file provides a way to sync sub dependencies for every developer who cloned the repo. Otherwise `"chai": "^4.2.0",` kinda dependencies cause inconsistency between developers.

Reply:
We use NPM instead of yarn here. if anything, the problem is the versions are not pinned. I'll pin them later on.

Reply:
actually, it pins submodules especially, anyway I've removed and added to `.gitignore`